x-pack/metricbeat/module/aws/s3_request/_meta/fields.yml

```diff
@@ -0,0 +1,74 @@
+- name: s3_request
+  type: group
+  description: >
+    `s3_request` contains request metrics that were scraped from AWS CloudWatch which contains monitoring metrics sent by AWS S3.
+  release: beta
+  fields:
+    - name: bucket.name
+      type: keyword
+      description: >
+        Name of a S3 bucket.
+    - name: all_requests
+      type: long
+      description: >
+        The total number of HTTP requests made to an Amazon S3 bucket, regardless of type.
+    - name: get_requests
+      type: long
+      description: >
+        The number of HTTP GET requests made for objects in an Amazon S3 bucket.
+    - name: put_requests
+      type: long
+      description: >
+        The number of HTTP PUT requests made for objects in an Amazon S3 bucket.
+    - name: delete_requests
+      type: long
+      description: >
+        The number of HTTP DELETE requests made for objects in an Amazon S3 bucket.
+    - name: head_requests
+      type: long
+      description: >
+        The number of HTTP HEAD requests made to an Amazon S3 bucket.
+    - name: post_requests
+      type: long
+      description: >
+        The number of HTTP POST requests made to an Amazon S3 bucket.
+    - name: select_requests
```

Comment:
Here maybe a `aws.s3_request.select.total` and `aws.s3_request.select.scanned.bytes`, `aws.s3_request.select.returned.bytes`